### PR TITLE
Fix for custom fields selection feature.

### PR DESCRIPTION
**Pull request by hawk-lord**
Fixes (no number).

- [x] This PR is compliant with the contributing guidelines (if not, please describe why): code is fully documented and if possible a .md file is made.
- [x] The PR text includes a **detailed explanation** (more than 50 chars)
- [x] I have thoroughly tested my contribution.

mcCombinations did not check with the settings if any custom credential fields had been selected before filling in credentials.

Also added two new combinations with one field each to work with multi-form pages such as vk.com.

Tested with:
- https://vk.com/
- https://account.sydsvenskan.se/register?appId=sydsvenskan.se
- http://www.ne.se/account/skapa-konto/privat-m%C3%A5nad
- https://trafikverket.ineko.se/se/register

### CODE REVIEW

File: chrome_extension/vendor/mooltipass/mcCombinations.js [hunk 1]
```diff
@@ -494,23 +494,49 @@ mcCombinations.prototype.possibleCombinations = [
 		maxfields: 1,
 		score: 0,
 		autoSubmit: false
+	},
+	{
+		combinationId: 'form001',
```
Comment (limpkin) [inline chrome_extension/vendor/mooltipass/mcCombinations.js]: any reason you added this? this represents quite a major change to our detection algo.

Comment (hawk-lord) [inline chrome_extension/vendor/mooltipass/mcCombinations.js]: Otherwise it would not find fields that are present in different forms.

Comment (hawk-lord) [inline chrome_extension/vendor/mooltipass/mcCombinations.js]: Is that not acceptable?

Comment (limpkin) [inline chrome_extension/vendor/mooltipass/mcCombinations.js]: it looks to me that you're completely changing the way we detect forms with this new maxfields value and scorepermatch value... so I'm afraid this is not acceptable.
Such drastic changes to our detection algo have massive repercussions.

Comment (hawk-lord) [inline chrome_extension/vendor/mooltipass/mcCombinations.js]: The present algorithm won't find any combination for fields such as first name and last name on vk.com, but it is dependent on found combinations to fill in the fields.

Comment (limpkin) [inline chrome_extension/vendor/mooltipass/mcCombinations.js]: then I'm not sure to follow you: on vk.com i'm actually able to select any login field with the "select custom credential fields" feature.
The goal of this feature is to actually override the complete mccombinations logic.

**Pull request closed without merging**